chutney/ui/src/app/modules/campaign/components/create-campaign/campaign-edition.component.ts

```diff
@@ -100,6 +105,12 @@ export class CampaignEditionComponent implements OnInit, OnDestroy {
         this.submitted = false;
         this.loadEnvironment();
         this.loadAllScenarios();
+        this.datasetService.findAll().subscribe((res: Array<Dataset>) => {
+            this.datasets = res.map(dataset => {
+                return {"id": dataset.name, "text": dataset.name}
+            });
+        });
+        this.dropdownDatasetSettings = {idField: "id",textField: "text", singleSelection: true, allowSearchFilter: true, enableCheckAll: false, closeDropDownOnSelection: true}
```

Comment:
- You can override only some properties and don't re-define default values.
- So what you need to change in your case is only  `singleSelection`.  `this.dropdownDatasetSettings.singleSelection = true`
- I prefer to don't close dropdownon selection ( keep default value of `closeDropDownOnSelection `which is false )